x-pack/legacy/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/index.tsx

```diff
@@ -33,7 +34,7 @@ export class ServiceIntegrations extends React.Component<Props, State> {
   static contextType = CoreContext;
   public state: State = { isPopoverOpen: false, activeFlyout: null };
 
-  public getPanelItems = memoize((mlAvailable: boolean) => {
+  public getPanelItems = memoize((mlAvailable: boolean | undefined) => {
```

Comment:
Why is this memoized?

Reply:
I guess because it's called in the render method and was deemed to be slow. Probably not necessary though.